packages/player-react/src/index.tsx

```diff
@@ -158,10 +215,10 @@ export function Player({
       >
         <div className="p-relative">
           <revideo-player
-            ref={playerRef}
+            ref={playerRef as LegacyRef<HTMLDivElement>}
```

Comment:
Interesting that we need to use LegacyRef here. What's the difference between the normal `RefObject` or `Ref` and this type?

Reply:
actually we don't need the type assertion at all